packages/babel-preset-latest/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-preset-latest
+
+> Babel preset including es2015, 2016, 2017.
```

Comment:
I think we'll want to explain what "latest" means here. I'm not sure what we want to put yet though. We can change it after merging too.